lib/OpenQA/WebSockets/Server.pm

```diff
@@ -279,6 +284,11 @@ sub setup {
     # start worker checker - check workers each 2 minutes
     Mojo::IOLoop->recurring(120 => \&_workers_checker);
 
+    my $service = OpenQA::IPC->ipc->{bus}->get_service("org.opensuse.openqa.Scheduler");
+    my $scheduler = $service->get_object("/Scheduler", "org.opensuse.openqa.Scheduler");
+
+    $scheduler->connect_to_signal("JobsAvailable" => \&jobs_available);
```

Comment:
yes, but there is no guarantee it already connected to DBUS when websocket starts